mongodbatlas/resource_mongodbatlas_event_trigger.go

```diff
@@ -0,0 +1,520 @@
+package mongodbatlas
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/go-test/deep"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+	"github.com/mwielbut/pointy"
+	"github.com/spf13/cast"
+	"go.mongodb.org/realm/realm"
+)
+
+const (
+	errorEventTriggersCreate  = "error creating MongoDB EventTriggers (%s): %s"
+	errorEventTriggersUpdate  = "error updating MongoDB EventTriggers (%s)%s: %s"
+	errorEventTriggersRead    = "error reading MongoDB EventTriggers (%s)%s: %s"
+	errorEventTriggersDelete  = "error deleting MongoDB EventTriggers (%s)%s: %s"
+	errorEventTriggersSetting = "error setting `%s` for EventTriggers(%s)%s: %s"
+)
+
+func resourceMongoDBAtlasEventTriggers() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceMongoDBAtlasEventTriggersCreate,
+		Read:   resourceMongoDBAtlasEventTriggersRead,
+		Update: resourceMongoDBAtlasEventTriggersUpdate,
+		Delete: resourceMongoDBAtlasEventTriggersDelete,
+		Importer: &schema.ResourceImporter{
+			State: resourceMongoDBAtlasEventTriggerImportState,
+		},
+		Schema: map[string]*schema.Schema{
+			"project_id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"app_id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"type": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: validation.StringInSlice([]string{"DATABASE", "AUTHENTICATION", "SCHEDULED"}, false),
```

Comment:
Ah you are right, SCHEDULED is not in the doc is it?  Is this what you mean when you said the docs were wrong?

Reply:
Yes and the response, the response was different than the docs, the response returns almost the same as the request with additional field like `function_name` and id

Reply:
Good to know, thanks!